index.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 const fs = require('fs');
 const pify = require('pify');
+
 const fsP = pify(fs);
 const fsReadP = pify(fs.read, {multiArgs: true});
 
+function createBuffer(len) {
+	if (Buffer.alloc) {
+		return Buffer.alloc(len);
+	}
+
+	// TODO: remove when when Node.js 6 is target (needed for Node.js < 5.10.0)
```

Comment:
[According to the docs](https://nodejs.org/api/buffer.html#buffer_class_method_buffer_alloc_size_fill_encoding), it is only available from version `5.10.0` on.
I think `Buffer.from` was backported to Node.js 4, but `Buffer.alloc` not.